pkg/kubelet/images/puller.go

```diff
@@ -31,6 +31,8 @@ type pullResult struct {
 	imageSize    uint64
 	err          error
 	pullDuration time.Duration
+	// pullCredentialsHash of successful pull with auth, nil if no auth used or error
```

Comment:
This comment also does not add a lot of value. Plus, you cannot set strings to nil in Go.

Reply:
Addressed